cli/dcoscli/cluster/main.py

```diff
@@ -254,7 +254,10 @@ def _user_cert_validation(cert_str):
     msg = "SHA256 fingerprint of cluster certificate bundle:\n{}".format(
             pp_fingerprint)
 
-    return confirm(msg, False)
+    if confirm(msg, False):
```

Comment:
Why not leave this code as it was and push the `raise` inside `confirm()`?

Reply:
I'll have to check other usages to see if it doesn't break things. Maybe in some cases non-confirming would be valid? Eg. asking for an optional thing.

Reply:
There are only 2 other usages of it. If I change the `confirm()` function to raise an exception, I'd also have to remove the return value as it'd become redundant (no exception raised means it was confirmed).

Should I do this? I find the current way to be more flexible, as explained above.

Reply:
as discussed in person, I'll leave it to return a boolean, and leave it to the caller what to do with it.